packaging/bootc/Containerfile.rhel

```diff
@@ -0,0 +1,22 @@
+FROM registry-proxy.engineering.redhat.com/rh-osbs/rhel9-rhel_bootc:rhel-9.4
+ADD etc etc
+RUN rm -rf /opt && \
+    mkdir -p /opt/cni
+RUN dnf install -y microshift
+RUN ln -s microshift.service /usr/lib/systemd/system/default.target.wants
+RUN dnf install -y flightctl-agent
+RUN ln -s /usr/lib/systemd/system/podman.socket /usr/lib/systemd/system/multi-user.target.wants/
```

Comment:
Now that we've switched to `exec`'ing to podman instead of using the lib, do we still need this?

Reply:
Good one, I totally missed that change.